Framework/PythonInterface/mantid/geometry/src/Exports/ComponentInfo.cpp

```diff
@@ -1,28 +1,31 @@
 #include "MantidGeometry/Instrument/ComponentInfo.h"
 #include "MantidKernel/Quat.h"
 #include "MantidKernel/V3D.h"
+#include "MantidPythonInterface/kernel/Converters/VectorToNDArray.h"
+#include "MantidPythonInterface/kernel/Converters/WrapWithNumpy.h"
+
 #include <boost/python/class.hpp>
 #include <boost/python/copy_const_reference.hpp>
+#include <boost/python/handle.hpp>
 #include <boost/python/list.hpp>
 #include <boost/python/return_value_policy.hpp>
 
 using Mantid::Geometry::ComponentInfo;
 using Mantid::Kernel::Quat;
 using Mantid::Kernel::V3D;
 using namespace boost::python;
+namespace Converters = Mantid::PythonInterface::Converters;
 
-// Helper function for converting std::vector<size_t> to a Python list
+// Helper function to call a converter for std::vector<size_t> to a Python list
 boost::python::list createList(std::vector<size_t> items) {
-  // Create the result list
-  boost::python::list result;
+  // Create a list to populate
+  boost::python::list dataAsList;
 
-  // Populate the Python list
-  std::vector<size_t>::iterator it;
-  for (it = items.begin(); it != items.end(); ++it) {
-    result.append(*it);
-  }
+  // Store the data
+  dataAsList.append(object(handle<>(
```

Comment:
Is this correct? I would have thought this gives you a list of vectors?

Have you considered exposing this as a `numpy` array, as done, e.g., here https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/mantid/api/src/Exports/MatrixWorkspace.cpp#L271?

Reply:
I think you are correct - I'll change it to be a numpy array.